go.mod

```diff
@@ -79,7 +79,7 @@ require (
 	github.com/prometheus/common v0.9.1
 	github.com/prometheus/tsdb v0.8.0 // indirect
 	github.com/rancher/dynamiclistener v0.2.1-0.20200418023342-52ede5ec9234
-	github.com/rancher/kontainer-engine v0.0.4-dev.0.20201120185903-87b9f0bb2861
+	github.com/rancher/kontainer-engine v0.0.4-dev.0.20201106160106-4442123b5eaa
```

Comment:
Now that I think about it, it would be safer if you pull in the reverted KE changes just in case, rather than reverting this. If there is another KE change down the line we don't want any accidental things to pop up that the revert-commit introduced.